tools.py

```diff
@@ -112,8 +112,8 @@ def git_add_and_pr(self, gem, matlab_output):
 
     def check_dependencies(self):
         # Check Matlab version
-        cmd = sp.check_output(['/usr/local/bin/matlab', '-nodisplay -nosplash -nodesktop -r', '"disp(version); quit"'])
-        m_version = ' '.join(cmd.decode('utf-8').split()[-1:]).replace('(','').replace(')','')
+        cmd = sp.check_output(['/usr/local/bin/matlab', '-nodisplay -nosplash -nodesktop -batch', '"disp(version); quit"'])
+        m_version = re.findall(r'([R]\d{4}\w+)',cmd.decode('utf-8'))[-1]
```

Comment:
A regular expression was indeed the way to go!

I propose taking as the version everything between `(` and `)` as long as the first letter is `R`, focusing on the first match with index `0`.

```
m_version = re.findall(r'\((R.+)\)',cmd.decode('utf-8'))[0]
```

Reply:
@mihai-sysbio I wouldn't recommend to include the parentheses in the regular expression as the version string is really unstable in its format, the most regular pattern that I could find was the already implemented one. Here you have an example on how the position of the parenthesis changes across versions: case [`version_ID + (R+year) + updateID`](https://github.com/SysBioChalmers/ecModels/runs/1939053091?check_suite_focus=true#step:3:9), case [`(version_ID) + Ryear`](https://github.com/MetabolicAtlas/ecModels/runs/1942281222?check_suite_focus=true#step:3:38).

But I will incorporate your suggestion about aiming to the firs element instead :)

Reply:
Nice!, it looks like the PR is ready to be merged.